Userland/Libraries/LibAudio/WavWriter.cpp

```diff
@@ -50,9 +50,9 @@ ErrorOr<void> WavWriter::write_samples(ReadonlySpan<Sample> samples)
             u8 left = static_cast<u8>((sample.left + 1) * scale);
             u8 right = static_cast<u8>((sample.right + 1) * scale);
             TRY(m_file->write_value(left));
-            TRY(m_file->write_value(right));
+            if (m_num_channels==2) TRY(m_file->write_value(right));
```

Comment:
WAV can have even more than two channels. Let's make this a >= 2 and worry about further channels later. (but at least a 3 channel file won't be mono)